lib/remote/jsonrpcconnection-pki.cpp

```diff
@@ -53,7 +53,13 @@ Value RequestCertificateHandler(const MessageOrigin::Ptr& origin, const Dictiona
 
 	String cn = GetCertificateCN(cert);
 
-	bool signedByCA = VerifyCertificate(cacert, cert);
+	bool signedByCA;
+
+	try {
+		signedByCA = VerifyCertificate(cacert, cert);
+	} catch (const std::exception& ex) {
+
+	}
```

Comment:
Is the `catch` block empty on purpose? If so can you add a short code comment about the swallowed exception?